test/test_manifest.json

```diff
@@ -1037,6 +1037,15 @@
        "lastPage": 1,
        "type": "eq"
     },
+    {  "id": "xfa_issue14315",
+      "file": "pdfs/xfa_issue14315.pdf",
+      "md5": "e239710875f9dea72fd94c20672a29e1",
+      "link": true,
```

Comment:
Given that the PDF document looks really small/simple, and looks to have created to be a good reduced test-case, in this case I'd suggest simply adding the document directly to the repository instead.